Src/Newtonsoft.Json/Newtonsoft.Json.Portable.csproj

```diff
@@ -136,11 +143,14 @@
     <Compile Include="Linq\JsonPath\ScanFilter.cs" />
     <Compile Include="Linq\JsonPath\ScanMultipleFilter.cs" />
     <Compile Include="Linq\JToken.cs" />
+    <Compile Include="Linq\JToken.Async.cs" />
     <Compile Include="Linq\JTokenEqualityComparer.cs" />
     <Compile Include="Linq\JTokenReader.cs" />
```

Comment:
Unless I missed something, there's nothing that can ever hit on async I/O since its all in-memory and hence nothing that could benefit from being done async. A few methods could be optimised such as returning CompletedTask directly from FlushAsync and having WriteValueAsync inline their work, but the improvement over the base is likely marginal, especially since we still have to guard against a subclass that did something else in Flush, etc.